_docs-sources/iac/reference/index.md

```diff
@@ -1,3 +1,7 @@
 # Library Reference
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lacus viverra vitae congue eu. Aliquam eleifend mi in nulla posuere sollicitudin. Imperdiet proin fermentum leo vel orci porta. Proin fermentum leo vel orci porta non pulvinar. Hac habitasse platea dictumst quisque sagittis purus sit. Malesuada bibendum arcu vitae elementum curabitur. Ultrices eros in cursus turpis massa tincidunt. Molestie ac feugiat sed lectus. Donec ac odio tempor orci dapibus ultrices in iaculis. Id ornare arcu odio ut sem. Condimentum lacinia quis vel eros donec ac odio. Enim eu turpis egestas pretium aenean pharetra magna ac placerat. Vulputate sapien nec sagittis aliquam malesuada. Ipsum consequat nisl vel pretium. Ipsum a arcu cursus vitae congue. Sit amet nisl purus in. Et ligula ullamcorper malesuada proin libero nunc consequat interdum.
+The Library Reference serves as the definitive source for all of the Modules and Services within the Gruntwork Infrastructure as Code Library. This comprehensive reference provides a dedicated page for each module and service providing descriptions, detailed information on input and output variables, and sample code to help you get started.
```

Comment:
"definitive source" sounds strange to me. Maybe definitive reference? index? documentation? IDK, words are hard

Reply:
I think `index` does sound better. Reading it back to myself, `source` would not be the right word anyway as GitHub is really the definitive source here, and I think it sounds a bit odd stating reference again.

`The Library Reference serves as the definitive index for all of the Modules and Services...` seems to work